### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#50945]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Authenticated 22-1990e Transfer of Entitlement (TOE) Form Updates

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The "Start your application without signing in" is outside the card.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Move the "Start your application without signing in" inside the card.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by btruongafs** [comment]
The "Start your application without signing in" link is purposefully outside of the card like to encourage users to focus and take the desired call to action to sign in before starting the application. This has been reviewed as a non-issue by our Product Owners Tammy Hurley and Eddie Ricks.

**Comment by shiragoodman** [comment]
Thanks @btruongafs . I understand that this may not be an issue for your Product Owners with regards to your product, however this is still an issue from the VA.gov Platform perspective because this doesn't align with our standards set for the VA.gov site. I have reopened this ticket and it will remain open until the issue is addressed. If you have questions or concerns about this, please let me know and we can discuss further.

cc @allison0034 @humancompanion-usds
